src/odinapi/database/level2db.py

```diff
@@ -326,7 +328,8 @@ def get_measurements(self, products,
                          min_altitude=None, max_altitude=None,
                          min_pressure=None, max_pressure=None,
                          start_time=None, end_time=None, areas=None,
-                         fields=None):
+                         fields=None, min_scanid=None,
+                         document_limit=HARD_LIMIT):
```

Comment:
I don't understand something: why can't you call it limit and offset, like in the other endpoints? Is it to clarify that this is different from what limit and offset/min_scanid means in this endpoint (if so, is it really different enough for a new name?) or is it something to do with Swagger perhaps? To me, new names seems more confusing than useful, but if you have a good reason I don't mind.

Reply:
Also, if we don't want the user to set document_limit, does it actually need to be a parameter to the endpoint for any other purpose than testing?

Reply:
this is due to swagger.

Reply:
Ok, sounds all right then. Still not sure document_limit actually needs to be a parameter to the endpoint though. But if it makes testing easier, it might be worth it?

Reply:
it does not need to be a parameter. but it makes testing easier/faster. for now I keep it as a parameter.